specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.generatedetailedcostreport.json

```diff
@@ -0,0 +1,407 @@
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "GenerateDetailedCostReportByBillingProfileAndInvoiceId": {
+            "$ref": "./examples/GenerateDetailedCostReportByBillingProfileAndInvoiceId.json"
+          },
+          "GenerateDetailedCostReportBySubscriptionAndTimePeriod": {
+            "$ref": "./examples/GenerateDetailedCostReportBySubscriptionAndTimePeriod.json"
+          },
+          "GenerateDetailedCostReportByCustomerAndTimePeriod": {
+            "$ref": "./examples/GenerateDetailedCostReportByCustomerAndTimePeriod.json"
+          },
+          "GenerateDetailedCostReportByBillingProfileAndInvoiceIdAndCustomerId": {
+            "$ref": "./examples/GenerateDetailedCostReportByBillingProfileAndInvoiceIdAndCustomerId.json"
+          },
+          "GenerateDetailedCostReportByBillingAccountLegacyAndBillingPeriod": {
+            "$ref": "./examples/GenerateDetailedCostReportByBillingAccountLegacyAndBillingPeriod.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeUsageDetailsParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GenerateDetailedCostReportDefinition"
+            },
+            "description": "Parameters supplied to the Create detailed cost report operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request processing completed.",
+            "schema": {
+              "$ref": "#/definitions/GenerateDetailedCostReportOperationResult"
+            }
+          },
+          "202": {
+            "description": "Accepted. Request will be processed. Use the Location header to check the status.",
+            "headers": {
+              "Location": {
+                "description": "The URL to check the result of the asynchronous operation.",
+                "type": "string"
+              },
+              "Azure-Consumption-AsyncOperation": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
```

Comment:
what's difference between `Azure-AsyncOperation` and `Azure-Consumption-AsyncOperation`?

Reply:
The both return the same value.
We have been offering both headers for this service for some time, so am maintaining consistency here.